internal/stubserver/stubserver.go

```diff
@@ -238,6 +238,7 @@ func (ss *StubServer) Stop() {
 	for i := len(ss.cleanups) - 1; i >= 0; i-- {
 		ss.cleanups[i]()
 	}
+	ss.cleanups = nil
```

Comment:
Hmm ... interesting.

Did you have to do this because you are calling `Stop` twice?

Reply:
There are no issues while calling `Close` multiple times. I was looking at the code to verify the same and saw that we could deallocate the slice to fee memory and ensure the cleanup funcs are run only once.

I can revert the change if its not needed.

Reply:
No, this is a good change to do anyways.